include/alpaka/queue/cpu/ICpuQueue.hpp

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2019 Axel Huebl, Benjamin Worpitz, Matthias Werner
+ *
+ * This file is part of Alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+namespace alpaka
+{
+    namespace event
+    {
+        class EventCpu;
+    }
+}
+
+namespace alpaka
+{
+    namespace queue
+    {
+        namespace cpu
+        {
+
+            //#############################################################################
+            //! The CPU queue interface
+            class ICpuQueue
+            {
+            public:
+                //-----------------------------------------------------------------------------
+                ICpuQueue() = default;
```

Comment:
Do we really need the explicitly default default construcor? I think this is not necessary for an abstract interface class.